### ISSUE TITLE
[Question] Kubernetes Deployment

### ISSUE THREAD

**Issue by serenewaffles** [blakeblackshear/frigate#913]
If I use Helm to deploy to Kubernetes, am I taking advantage of the nature of the cluster? That is will Frigate spawn pods for object detection (or other tasks) that can be spread around the cluster?

**Comment by blakeblackshear** [comment]
No. The overhead of sending things around the cluster would not be worth it. Frigate uses shared memory heavily, so all the processes need to be in the same container. I would recommend scaling out by running multiple instances of the container and splitting your cameras up among them. It really shouldn't be necessary unless you have a lot of cameras. Make sure you check open issues on the chart repo.

**Comment by serenewaffles** [comment]
Ok, thanks for the information. I've set up my cluster using RPi4 units without any sort of USB detection dongles, so I was hoping to leverage the clusterized nature by perhaps spawning a pod per camera automatically. I imagine the main pod would just coordinate the web frontend while letting the other pods handle detection, video saving, etc.

In any case, I'll be keeping an eye on this project. So far, this is the most intuitive NVR with object detection I've used.
